docker-compose.yml

```diff
@@ -1,16 +1,25 @@
-version: '2'
+version: '3'
 
 services:
   nuxt:
     build:
       context: .
+    depends_on:
+    - db
+    - ipfs
     ports:
       - "3000:3000"
     volumes:
-      - ./:/usr/src/app
-    links:
-      - db
-      - ipfs
+      - ./.nuxt:/usr/src/app/.nuxt
+      - ./assets:/usr/src/app/assets
+      - ./build:/usr/src/app/build
+      - ./components:/usr/src/app/components
+      - ./layouts:/usr/src/app/layouts
+      - ./migrations:/usr/src/app/migrations
+      - ./pages:/usr/src/app/pages
+      - ./plugins:/usr/src/app/plugins
+      - ./server:/usr/src/app/server
+      - ./static:/usr/src/app/static
```

Comment:
Why we want this?

I assume you want to keep the node_modules inside docker and outside docker is separated.

This setup have potential when developer adding a new folder and forgot to add here. I prefer the original setup.

Reply:
Map node_modules into docker is slow